executor/aggfuncs/set_with_memory_usage.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs
+
+const (
+	// Maximum average load of a bucket that triggers growth is 6.5.
+	// Represent as loadFactorNum/loadFactDen, to allow integer math.
+	loadFactorNum = 13
```

Comment:
Can we use the var defined in aggregate.go?

Reply:
No. It will cause cycle import.
I try to move the two var to package `set` and export it. PTAL.

Reply:
Maybe we can move it to util/hack.go

And add a comment for these 2 consts to describe that they are from the golang definition.